django_recaptcha/static/django_recaptcha/js/widget_v2_checkbox.js

```diff
@@ -0,0 +1,36 @@
+(function () {
+
+  function main() {
+    for (const captchaElement of document.querySelectorAll(".g-recaptcha")) {
+
+      const widgetUuid = captchaElement.getAttribute("data-widget-uuid");
```

Comment:
```suggestion
      const widgetUUID = captchaElement.getAttribute("data-widget-uuid");
```

Should use the acronym’s case